snuba/migrations/snuba_migrations/transactions/0016_transactions_add_group_ids_column.py

```diff
@@ -0,0 +1,86 @@
+from typing import List, Sequence, Tuple
+
+from snuba.clickhouse.columns import Array, Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+new_columns: Sequence[Tuple[Column[Modifiers], str]] = [
+    (Column("group_ids", Array(UInt(64))), "timestamp"),
+    (
+        Column(
+            "_group_ids_hashed",
+            Array(
+                UInt(64),
+                Modifiers(materialized="arrayMap(t -> cityHash64(t), group_ids)"),
+            ),
+        ),
```

Comment:
Why the hashed version ?
The group id is already a UInt64 so the hashed version is still the same type. THe bloom filter data structure already computes hashes (multiple of them) of the value.

Reply:
addressed 👍

Reply:
is there anything I need to do to explicitly enable bloom filter?